### DISCUSSION TITLE
How to fetch mysql timestamp column?

### DISCUSSION THREAD

**Discussion by joyatcloudfall** [fastapi/fastapi#6148] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
from fastapi import Depends, FastAPI, HTTPException
from sqlalchemy.orm import Session
import uvicorn
from sql_app import crud, models, schemas
from sql_app.database import SessionLocal, engine

models.Base.metadata.create_all(bind=engine)

app = FastAPI()


# Dependency
def get_db():
    db = SessionLocal()
    try:
        yield db
    finally:
        db.close()



@app.get("/customers/{cid}", response_model=schemas.Customer)
def read_user(cid: int, db: Session = Depends(get_db)):
    db_user = crud.get_customer(db, cid=cid)
    if db_user is None:
        raise HTTPException(status_code=404, detail="Customer not found")
    return db_user

if __name__ == "__main__":

    uvicorn.run(app, port=7070)
```

### Description

I just use FastAPI for a few days. I would like to try to use sqlalchemy to fetch data from mysql. However, I got this error since the data type of createdTime and updatedTime in mysql is TIMESTAMP. Is there any way to convert datetime and timestamp?

```
pydantic.error_wrappers.ValidationError: 1 validation error for Customer
response -> createTime
  field required (type=value_error.missing)
```

models.py

```
from sqlalchemy import Column, ForeignKey, Integer, String, TIMESTAMP
from sqlalchemy.orm import relationship

from sql_app.database import Base

class Status(Base):
    __tablename__ = "status"

    statusId = Column(Integer,primary_key=True)
    statusName = Column(String, index=True)
    createdTime = Column(TIMESTAMP())
    updatedTime = Column(TIMESTAMP())

    customerState = relationship("Customer", back_populates="state")

class Customer(Base):
    __tablename__ = "customer"

    cid = Column(Integer, primary_key=True)
    customerName = Column(String, unique=True)
    statusId = Column(Integer, ForeignKey("status.statusId"))
    createdTime = Column(TIMESTAMP)
    updatedTime = Column(TIMESTAMP)

    state = relationship("Status", back_populates="customerState")
```

schemas.py

```
from datetime import datetime
from pydantic import BaseModel


class StatusBase(BaseModel):
    statusId: int
    statusName: str


class StatusCreate(StatusBase):
    pass


class Status(StatusBase):
    pass
    class Config:
        orm_mode = True


class CustomerBase(BaseModel):
    customerName: str


class CustomerCreate(CustomerBase):
    pass


class Customer(CustomerBase):
    cid: int
    state: Status
    createTime: datetime
    updatedTime: datetime

    class Config:
        orm_mode = True
```

crud.py

```
from sqlalchemy.orm import Session

from sql_app import models


def get_customer(db: Session, cid: int):
    return db.query(models.Customer).filter(models.Customer.cid == cid).first()


def get_customer_by_email(db: Session, customerName: str):
    return db.query(models.Customer).filter(models.Customer.customerName == customerName).first()
```

### Operating System

Windows

### FastAPI Version

0.79.0

### Python Version

3.9.10

**Comment by Jedore** [comment]
You should descirbe more detailed.

**Comment by joyatcloudfall** [comment]
I have table customer and status in mysql. When I read customer without createdTime and updatedTime, I can get customer via API. 

```
class Customer(CustomerBase):
    cid: int
    state: Status

    class Config:
        orm_mode = True
```

**Comment by Jedore** [comment, marked as answer]
@joyatcloudfall  I test and find your spelling error. In `schemas.Customer` is `createTime`, but in `models.Customer` is `createdTime`, the difference is a `d`.

**Comment by joyatcloudfall** [comment]
Indeed. Sorry, that's my bad. I should have checked the code more carefully. Thanks anyway.

**Comment by tiangolo** [comment]
Thanks for the help here @Jedore  ! 👏  🙇 

Thanks for reporting back and closing the issue @joyatcloudfall  👍
